index.bs

```diff
@@ -550,12 +550,12 @@ When this method is invoked, the user agent MUST execute the following algorithm
     object=]'s [=relevant settings object=]. If |callerOrigin| is an [=opaque origin=], return a {{DOMException}} whose name is
     "{{NotAllowedError}}", and terminate this algorithm.
 
-1. Let |effectiveDomain| be the |callerOrigin|'s [=effective domain=]. If |effectiveDomain| is null,
-    then return a {{DOMException}} whose name is "{{SecurityError}}" and terminate this algorithm.
-<!-- PLACEHOLDER - https://github.com/w3c/webauthn/issues/474
-    Note: An [=effective domain=] MUST be a [=valid domain=]. Formats other than [=domain=] -- i.e., [=ipv4 address=],
+1. Let |effectiveDomain| be the |callerOrigin|'s [=effective domain=].
+    If [=effective domain=] is not a [=valid domain=], then return a
+    {{DOMException}} whose name is "{{SecurityError}}" and terminate this algorithm.
+
+    Note: [=host=] formats other than [=domain=] -- i.e., [=ipv4 address=],
```

Comment:
this is somewhat jarring in that the alg step itself is in terms of "domain", not "host".
suggest:
>Note: An [=effective domain=] may resolve to a [=host=], which can be represented in various manners, such as [=domain=], [=ipv4 address=], [=ipv6 address=], [=opaque host=], or [=empty host=]. Only the [=domain=] format of [=host=] is allowed here.